src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtAreaOfLawEndpointTest.java

```diff
@@ -70,9 +71,18 @@ public void shouldReturnNotFoundWhenCourtDoesNotExist() {
     }
 
     /************************************************************* PUT request tests section. ***************************************************************/
+    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
     @Test
     public void shouldUpdateCourtAreaOfLaw() throws JsonProcessingException {
-        final List<AreaOfLaw> currentCourtAreasOfLaw = getCurrentCourtAreasOfLaw();
+        final List<AreaOfLaw> currentCourtAreasOfLaw = getCurrentCourtAreasOfLaw()
+            .stream()
+            .map(aol -> {
+                AreaOfLaw newAol = new AreaOfLaw();
+                newAol.setId(aol.getId());
+                newAol.setName(aol.getName());
+                return newAol;
+            })
+            .collect(Collectors.toList());
```

Comment:
@Ayisha-Sharjeel Why do we need this new change here? Previously this test is adding a new area of law. Now this new change also wipe out those new additional fields before adding a new area of law before calling the PUT endpoint. Sorry just curious what is the purpose of that